src/pages/home/report/ReportActionsView.js

```diff
@@ -169,6 +178,17 @@ class ReportActionsView extends React.Component {
             return true;
         }
 
+        const policy = this.props.policies[`${ONYXKEYS.COLLECTION.POLICY}${this.props.report.policyID}`];
```

Comment:
Given this component only is interested in one policy, why don't we pass just that policy as prop?

Reply:
@iwiznia 
The policy will come as undefined in case of group chats and personal chats. Didn't want to pass undefined values through props.
And this is consistent with the existing props structure in ReportScreen.js. That's why request to have it this way.

https://github.com/Expensify/App/blob/555f761cecc633ea423be87d985731ee17527f33/src/pages/home/ReportScreen.js#L287-L293

Reply:
Ok Got it. Maybe we can change previous state. Sorry for the inconvience and churn @hungvu193